README.md

```diff
@@ -52,7 +52,7 @@ You can run the example either using `mpirun` or `horovodrun`.
 TF_HVD_DIR=/home/<USER>/tf_hvd && \
 PYTHONPATH=${TF_HVD_DIR}/lib && \
 PATH=${TF_HVD_DIR}/bin:${PATH} \
-mpirun --allow-run-as-root -bind-to none -map-by slot -x NCCL_DEBUG=INFO -x LD_LIBRARY_PATH -x PATH -x NCCL_SOCKET_IFNAME=^lo,docker0 -mca pml ob1 -mca btl ^openib -mca btl_tcp_if_exclude lo,docker0 \
```

Comment:
I'd leave the `-x LD_LIBRARY_PATH` since it's needed in some situations, unless it's causing failures.

Reply:
okay. I'll make a note of it